website/src/docs/hotchocolate/defining-a-schema/mutations.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Mutations"
+---
+
+import { ExampleTabs } from "../../../components/mdx/example-tabs"
+
+The mutation type in GraphQL is used to mutate/change data. This means that when we are doing mutations, we are causing side-effects to the system.
+
+GraphQL defines mutations as top-level fields on the mutation type. Meaning only the fields on the mutation root type itself are mutations. Everything that is returned from a mutation field represents the changed state of the server.
+
+```sdl
+type Mutation {
+  addBook(input: AddBookInput!): AddBookPayload!
+  publishBook(input: PublishBookInput!): PublishBookPayload!
+}
+```
+
+Clients can execute one or more mutations through the mutation type.
+
+```graphql
+mutation {
+  addBook(input: { title: "C# in depth" }) {
+    book {
+      id
+      title
+    }
+  }
+  publishBook(input: { id: 1 }) {
+    book {
+      publishDate
+    }
+  }
+}
+```
+
+Each of these mutations is executed serially one by one whereas their child selection sets are executed possibly in parallel since only the top-level mutation fields are allowed to have side-effects in GraphQL.
```

Comment:
```suggestion
Each of these mutations is executed serially one by one whereas their child selection sets are executed possibly in parallel since only top-level mutation fields (those directly under `mutation`) are allowed to cause side-effects in GraphQL.
```